pip/qsharp/_qsharp.py

```diff
@@ -59,6 +59,25 @@ def eval_file(path) -> None:
     return eval(f.read())
 
 
+def run(entry_expr, shots):
+    """
+    Runs the given Q# expressin for the given number of shots.
+    Each shot uses an independent instance of the simulator.
+
+    :param entry_expr: The entry expression.
+    :param shots: The number of shots to run.
+
+    :returns values: A list of results or runtime errors.
+
+    :raises QSharpError: If there is an error interpreting the input.
+    """
+
+    def callback(output):
+        print(output)
+
+    return _interpreter.run(entry_expr, shots, callback)
```

Comment:
This should use `get_interpreter()` to ensure it will initialize the value if it hasn't been yet:

```suggestion
    return get_interpreter().run(entry_expr, shots, callback)
```